compiler/testData/codegen/box/diagnostics/functions/tailRecursion/recursiveCallInInlineLambda.kt

```diff
@@ -1,13 +1,12 @@
 // KT-16549
-// IGNORE_BACKEND: JVM
-// IGNORE_FIR_DIAGNOSTICS_DIFF
+// IGNORE_BACKEND: JVM, JS
```

Comment:
Could you please keep old test for JS if there is no any similar one?

Reply:
The reason I had to mute it is a stack overflow, meaning, the non-IR JS backend doesn't optimize the call here. That's kind of the entire point of the test though...

Reply:
Yes, optimization is not working here. But it also tests that non-optimized case here is working as expected (and it's not optimized in wrong way)

Reply:
Looks like it's not neccessery if test would be fixed by https://github.com/JetBrains/kotlin/pull/4577#discussion_r712786380